packages/regl-worldview/src/commands/index.js

```diff
@@ -10,14 +10,14 @@ export { default as Command, SimpleCommand, makeCommand } from "./Command";
 
 // Primitives
 export { default as Arrows } from "./Arrows";
-export { default as Cones } from "./Cones";
-export { default as Cubes } from "./Cubes";
-export { default as Cylinders } from "./Cylinders";
+export { default as Cones, cones } from "./Cones";
+export { default as Cubes, cubes } from "./Cubes";
```

Comment:
Where/how are we hoping to reuse these exactly? What about just letting someone use `Cubes.command` instead of separately exporting `cubes`?

Reply:
Makes sense. Going to revert the changes. Thanks!